src/dbnode/storage/index/block.go

```diff
@@ -429,13 +429,9 @@ func (b *block) Query(
 
 func (b *block) queryWithSpan(
 	ctx context.Context,
-	cancellable *xresource.CancellableLifetime,
 	query Query,
 	opts QueryOptions,
-	results DocumentResults,
-	sp opentracing.Span,
-	logFields []opentracinglog.Field,
-) (bool, error) {
+	results DocumentResults) (bool, error) {
```

Comment:
nit: For consistency the majority of our indentation for function definitions usually has each argument with a trailing comma like this was before:
e.g.
```go
func (b *block) queryWithSpan(
  ctx context.Context,
  ...
  results DocumentResults,
) (bool, error) {
  // ...
}
```